app/views/application/_footer.html.slim

```diff
@@ -2,19 +2,19 @@
   .row
     .col-md-6
       h6
-        = t('layouts.footer.more.header')
-      = link_to t('layouts.footer.more.link'), about_home_index_path
+        = t('.more.header')
+      = link_to t('.more.link'), about_home_index_path
       br
       br
       | &copy;
       = " "
       = link_to "Hasso Plattner Institute", 'https://hpi.de/'
```

Comment:
HPI should be a translation, too. In German, we'll write "Hasso-Plattner-Institut". Further, those links should be configurable ideally, but this is something for another ticket ;).